rpcserver.go

```diff
@@ -953,8 +954,13 @@ func (r *rpcServer) ListAssets(ctx context.Context,
 	}
 
 	rpcAssets, err := r.fetchRpcAssets(
-		ctx, req.WithWitness, req.IncludeSpent, req.IncludeLeased,
+		ctx,
+		req.WithWitness,
+		req.IncludeSpent,
+		req.IncludeLeased,
+		req.MinAmount,
 	)
+
```

Comment:
extra newline

Reply:
What's the style guide here? No newlines after parentheses, but yes to newlines after curly braces? :thinking:

Reply:
it's specifically for when we do `if err != nil` right after a func call

even if the call is multiline we stich the `if err` statement right after